src/current/cockroachcloud/quickstart.md

```diff
@@ -99,7 +105,97 @@ The code sample uses the connection string stored in the environment variable `J
     BUILD SUCCESSFUL in 3s
     2 actionable tasks: 2 executed
     ~~~
+
+</section>
+<section class="filter-content" markdown="1" data-scope="node">
+
+Once you create a SQL user, the **Connect to cluster** dialog will show information about how to connect to your cluster.
+
+1. Select **General connection string** from the **Select option** dropdown.
+1. Open the **General connection string** section, then copy the connection string provided and save it in a secure location.
+
+    This Quickstart uses default certificates, so you can skip the Download CA Cert instructions.
+
+    {{site.data.alerts.callout_info}} 
+    The connection string is pre-populated with your username, password, cluster name, and other details. Your password, in particular, will be provided *only* once. Save it in a secure place (Cockroach Labs recommends a password manager) to connect to your cluster in the future. If you forget your password, a Cluster Administrator can reset it. Refer to: [Managing SQL users on a cluster](managing-access.html#manage-sql-users-on-a-cluster)
+    {{site.data.alerts.end}}
+
+## Configure the connection environment variable
+
+<div class="filters clearfix">
```

Comment:
Consider indenting the nested `<div>` and `<section>` tags to make it. clear that we are still within a top-level section.

Reply:
For some reason this makes everything render incorrectly